map_view/map-rotation/src/main/java/com/esri/samples/map_rotation/MapRotationSample.java

```diff
@@ -67,41 +66,26 @@ public void start(Stage stage) {
       String yourAPIKey = System.getProperty("apiKey");
       ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-      // create a slider with a range of 360 units and set it to 5 degrees
-      slider = new Slider(-180.0, 180.0, 5.0);
-      styleSlider();
-
-      // listen for the value in the slider to change
-      slider.valueProperty().addListener(e -> {
-        // rotate map view based on new value in slider
-        mapView.setViewpointRotationAsync(slider.getValue());
-      });
-
       // create a map with the streets basemap style
       map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
 
-      // enable slider when map view is done loading
-      map.addDoneLoadingListener(() -> {
-        if (map.getLoadStatus() == LoadStatus.LOADED) {
-          slider.setDisable(false);
-        } else {
-          Alert alert = new Alert(Alert.AlertType.ERROR, "Map Failed to Load!");
-          alert.show();
-        }
-      });
-
       // create a map view and set the map to it
       mapView = new MapView();
-      // disable keyboard navigation to focus on slider/compass map rotation interaction
-      mapView.setEnableKeyboardNavigation(false);
       mapView.setMap(map);
 
-      // create a compass to show the current heading when rotated
-      Compass compass = new Compass(mapView);
+      // create a compass and show the current heading
+      var compass = new Compass(mapView);
 
-      // clicking the compass sets the map's heading to 0.0 (north) and the compass is hidden
-      // add a listener to reset the slider when this happens
-      compass.setOnMouseClicked(e -> slider.setValue(0.0));
+      // create labels for instructions and map rotation
+      Label instructionsLabel = new Label("Press the A and D keys to rotate the map.");
+      instructionsLabel.setTextFill(Color.WHITE);
+      Label rotationLabel = new Label("Current rotation: ");
+      rotationLabel.setTextFill(Color.WHITE);
+
+      // update the rotation label when the compass rotates
+      rotationLabel.textProperty().bind(Bindings.createStringBinding(()-> {
+        return "Current map rotation: " + Math.round(compass.headingProperty().get()) + "º";
+      }, compass.headingProperty()));
```

Comment:
based on IntelliJ suggestion to replace statement lambda with expression lambda: good for readability
```suggestion
      rotationLabel.textProperty().bind(Bindings.createStringBinding(()-> 
        "Current map rotation: " + Math.round(compass.headingProperty().get()) + "º", compass.headingProperty()));
```